modules/gapi/samples/onevpl_infer_single_roi.cpp

```diff
@@ -345,7 +345,7 @@ int main(int argc, char *argv[]) {
         } else {
             cap = cv::gapi::wip::make_onevpl_src(file_path, source_cfgs);
         }
-        std::cout << "oneVPL source desription: " << cap->descr_of() << std::endl;
+        std::cout << "oneVPL source desrciption: " << cap->descr_of() << std::endl;
```

Comment:
desrciption -> description

Reply:
Fixed in 44dd15cc195acad5b2b41c33d2325afbfe00a2af